tests/support/bower.json

```diff
@@ -8,6 +8,7 @@
     "purescript-prelude": "1.1.0",
     "purescript-st": "1.0.0-rc.1",
     "purescript-partial": "1.1.2",
-    "purescript-newtype": "0.1.0"
+    "purescript-newtype": "0.1.0",
+    "purescript-generics-rep": "2.0.0"
```

Comment:
Should we update the rest of the versions of these while we're at it? Will probably need to tweak some tests, but may as well get it done, since the dependencies conflict with this (I think?).